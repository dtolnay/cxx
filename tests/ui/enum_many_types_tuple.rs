#[cxx::bridge]
mod ffi {
    enum Bad {
        A(i32, bool),
        B(i32),
    }
}

fn main() {}
