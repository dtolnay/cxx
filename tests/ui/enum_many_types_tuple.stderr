error: More than one unnamed field is not supported
 --> tests/ui/enum_many_types_tuple.rs:3:5
  |
3 | /     enum Bad {
4 | |         A(i32, bool),
5 | |         B(i32),
6 | |     }
  | |_____^
